server/controllers/review.js

```diff
@@ -0,0 +1,34 @@
+const Review = require("../models/Review");
+const Profile = require("../models/Profile");
+const asyncHandler = require("express-async-handler");
+
+exports.getReviews = asyncHandler(async (req, res) => {
+    const { id } = req.user;
```

Comment:
We will be fetching reviews for a sitter's profile and not the logged in user (considering passing the sitter's user/profile id as a route param)

Reply:
changed to using profileId param